scimath/units/length.py

```diff
@@ -71,7 +71,7 @@
 
 m = meter
 nm = nanometer
-um = micrometer
+µm = um = micrometer
```

Comment:
Probably worth adding a test that both `µm` (using MICRO SIGN) and `μm` (using GREEK SMALL LETTER MU) are parsed correctly with this definition. They are right now, but that could change if we move away from using `eval` at some point in the future.

Reply:
FWIW, and somewhat surprisingly, Unicode TR 25 says that GREEK SMALL LETTER MU is the preferred character here. (See section 2.5 of http://www.unicode.org/reports/tr25/)